fontir/src/ir.rs

```diff
@@ -695,10 +695,23 @@ impl GlyphPathBuilder {
 
     pub fn close_path(&mut self) -> Result<(), PathConversionError> {
         // Take dangling off-curves to imply a curve back to sub-path start
-        if let Some(last_move) = self.last_move_to {
+        if let Some(last_move) = self.last_move_to.take() {
```

Comment:
I had not thought of that, good point. I figured last_move_to should be reset to None after using the value in close_path, as it looked to me there is no longer a last move_to after the current path was closed. Anyways, right now the GlyphPathBuilder seems to be only ever used (and tested) to build single contour paths so it won't make a difference in practice. I'll revert that change.

Reply:
by the way, for SVG path I don't think we want to use this GlyphPathBuilder, because the latter seems to be designed to convert from a point-based path representation (as in UFO or Glyphs sources) to a segment-based path representation (kurbo BezPath). In SVG we already have segments or path drawing commands which mimic (with notable exception of arcs) the BezPath methods quite closely